index.html

```diff
@@ -51,5 +51,201 @@ <h3 class="connect">LET'S CONNECT</h3>
         </section>
       </div>
     </section>
+    <section class="work">
```

Comment:
- [ ] I like the color you have in your background :+1: , however, is a project requirement to follow the template as much as possible. Please let's fix this.

## Figma
![image](https://user-images.githubusercontent.com/28109626/148600207-18df9b97-2319-4469-ba14-07c3c4474b20.png)
## Your project
![image](https://user-images.githubusercontent.com/28109626/148600251-0c886263-e858-4c82-a590-07d052225021.png)